packages/rum-core/src/performance-monitoring/span-base.js

```diff
@@ -29,7 +29,16 @@ import { NAME_UNKNOWN, TYPE_CUSTOM } from '../common/constants'
 class SpanBase {
   // context
 
-  constructor(name = NAME_UNKNOWN, type = TYPE_CUSTOM, options = {}) {
+  constructor(name, type, options = {}) {
+    /**
+     * Check for undefined and empty string
```

Comment:
We should have caught this once we did this change recently, would you please add tests for falsy values to make sure default values are always used!